### PR TITLE
Incorporating CI pipeline into the fork staging

### PR DESCRIPTION
**Pull request by maasva**
### Description

Updated CI pipeline to build, test and style

### Requirements

Automating quality checks to ease the development burden. 

### Configurations

NA just make sure the Config files are up to date. 

### Related Issues

https://github.com/ForestForesight/ForestForesight/issues/10

### CODE REVIEW

File: .github/workflows/build_test.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+name: R
+
+on:
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  Building-testing-package:
+    runs-on: ubuntu-latest
+    # if: github.event.pull_request.merged == true || github.event.review.state == 'approved'
```
Comment (aminbksi) [inline .github/workflows/build_test.yml:10]: seems like an unnecessary comment. remove it plese

File: .github/workflows/styler_lintr.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 name: Improving Readability
 on:
   pull_request:
-    branches: [ "develop" ]
+    branches: 
+      - "develop"
+      - "main"
```
Comment (aminbksi) [inline .github/workflows/styler_lintr.yml:6]: are we keeping the main for styler?

File: config.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ AWS_BUCKET_NAME: "forestforesight-public"
 AWS_BUCKET_REGION: "eu-west-1"
 EARLIEST_DATA_DATE: "2021-01-01"
 DEFAULT_GROUNDTRUTH: "groundtruth6m"
-CODE_LOCATION: "D:/WWF/Repos/fork-1/ForestForesight"
+DATA_FOLDER: "C:/Kodingan3/FFdatas"
```
Comment (aminbksi) [inline config.yml:6]: can we also keep this folder empty when merging to develop, and maybe a comment that this should be empty string here

**Review by aminbksi — APPROVED** [review]
looks good, just requested a few minor changes

File: .github/workflows/styler_lintr.yml (hunk 2 above)
Comment (maasva) [reply .github/workflows/styler_lintr.yml:6]: Yeah, cannot hurt.

**Pull request merged**